### PR TITLE
login and signup validation form commited

### PR DESCRIPTION
**Pull request by BARNARDOHerve**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (Anguandia) [inline .vscode/launch.json:4]: What's this?

File: scripts/app.js [hunk 2]
```diff
@@ -4,31 +4,74 @@ function newUser() {
...
+function userLogin() {
     let email = document.getElementById("email").value;
     let password = document.getElementById("password").value;
+    let errorMessage = document.getElementById("errorMessage");
+    let atposition = email.indexOf("@");
+    let dotposition = email.lastIndexOf(".");
+
+    if (email == "") {
+        return errorMessage.innerHTML = "email is required"
+    } else if (atposition < 1 || dotposition < atposition + 2 || dotposition + 2 >= email.length) {
```
Comment (Anguandia) [inline scripts/app.js:51]: Nice logic

**Review by Anguandia — COMMENTED** [review]
Good work have you done anything on firebase?

**Review by ConstantineMaliza — COMMENTED** [review]
approved

File: scripts/app.js [hunk 3]
```diff
@@ -4,31 +4,74 @@ function newUser() {
...
+
+
     firebase.auth().createUserWithEmailAndPassword(email, password).then((user) => {
+            // alert("I cont");
             window.location.href = "../pages/login-in.html"
 
         })
         .catch((error) => {
-            alert();
+            alert(error);
```
Comment (Anguandia) [inline scripts/app.js:37]: Please use a better response displays

**Pull request merged**